tcms/bugs/tests/test_models.py

```diff
@@ -47,3 +48,11 @@ def test_no_notification_if_assignee_not_set(self, send_mail):
         BugFactory(assignee=None)
 
         self.assertFalse(send_mail.called)
+
+
+class TestBugTagPermissions(TestCase):
+    def test_add_bugtag(self):
```

Comment:
a better name for this would be something like `test_add_bugtag_permission_exists` or `test_add_permission_exists` (because we have bug tag in the class name), because this is what the test actually does. valid for the other test method as well